tendrl/node_agent/manager/__init__.py

```diff
@@ -95,6 +98,10 @@ def main():
     m = NodeAgentManager()
     m.start()
 
+    NS.ceph_integrations_sync_thread = \
```

Comment:
Please start this thread when

if NS.tendrl_context.sds_name == "ceph" and NS.tendrl_context.integration_id is not None: